packages/components/src/utils/test/rtl.test.js

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
We've a lot of inconsistency in the project, but the project guidelines are such that I would expect this file should be named `test/rtl.js`.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/testing-overview.md#folder-structure

Reply:
Ah, gotcha! Will make that update! 🤗